src/panels/Menu/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface TaskbarItemConfig {
+  id: string;
+  visible: boolean;
```

Comment:
Add comments decsribing what "visible" and "enabled" means? So that future us remember the difference?